src/stores/Blocks.js

```diff
@@ -36,6 +36,18 @@ let Blocks = {
     return state.filter(i => i.parent === parent)
   },
 
+  getDepth(state, block) {
+    let currentBlock = block;
+    let depth = 0
+
+    while (currentBlock || depth == 20) {
```

Comment:
Sorry, that was for while I was debugging it to prevent an infinite loop that crashed my browser. Now that the block is working though I should remove that, it seems unnecessary. Unless you think a guard like that is a best practice?